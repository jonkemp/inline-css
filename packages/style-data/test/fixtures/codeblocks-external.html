<!DOCTYPE html>
<html>
  <body>
    <h1><<Hi>></h1></body></html>
