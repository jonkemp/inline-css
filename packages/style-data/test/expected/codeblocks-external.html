<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <h1><<Hi>></h1>
  </body>
</html>
